wordpress-mobile/WordPress-iOS
# Issue/9787 fix location change

Fixes #9787 

To test:
 - Open a post where you didn't set location before
 - Go to Post Settings
 - Set a location
 - Close post settings
 - Check that the Update button is active
 - Tap on Update
 - Go to  post settings again
 - Remove the location
 - Close post settings
 - Check that the Update button is activated again.
